app/js/app.js

```diff
@@ -211,6 +213,22 @@ ddsApp.run(function($rootScope, $state, $stateParams, $window, $modalStack, $anc
         }
     });
 
+    $rootScope.$on('$stateChangeSuccess', function (event, current) {
+        // Default - set page focus to first h1
+        if (! current.preventFocus) {
+            // $timeout adds the function to the next angular digest cycle,
+            // waiting for the page to load before executing it
+            $timeout(function() {
+                var title = document.querySelector('h1');
+                // If anyone wants to set a tabindex manually, do not overwrite it
+                if (title && title.tabIndex < 0) {
```

Comment:
Because the [default value is `-1`](https://html.spec.whatwg.org/multipage/interaction.html#dom-tabindex), even though it has to be _explicitly set to `-1`_ to support `.focus()`… Ok, thanks, DOM API designers 😑